setup.py

```diff
@@ -34,11 +34,16 @@ def read(fname):
       tests_require=['pytest>=5.4.2',
                      'flake8>=3.7.2',
                      'gitpython>=3.1.2'],
+<<<<<<< HEAD
```

Comment:
🤔

Reply:
upsi. sorry, this branch is still heavily under development (thanks for finding this though). if you want to use it, currently only the ordinal approach works too (hoping to reconcile both approaches soon).

Reply:
> upsi. sorry, this branch is still heavily under development (thanks for finding this though). if you want to use it, currently only the ordinal approach works too (hoping to reconcile both approaches soon).

Nah, just looking through my github notifications and spotted this when the PR opened.